doc/man3/OpenSSL_version.pod

```diff
@@ -49,8 +49,6 @@ Deprecated:
 
 =head1 DESCRIPTION
 
-=head2 Macros
-
```

Comment:
I dislike the changes in this file. This is also an indication that your check is flawed with regards to sub-sections. Those you detected here *are* unique within each top-level section.

Reply:
You're right about the limitation in find-doc-nits, and I can work on fixing it.  I dislike the macros/functions subheads as a distinction. We don't do it anywhere else, and it's not common in other manpages (do ```man putchar``` for example).  HOWEVER, that's a discussion to have elsewhere, so I will undo the changes and fix the script.